packages/@okta/vuepress-site/docs/reference/api/oie-policies/index.md

```diff
@@ -0,0 +1,737 @@
+| status   | String  | `ACTIVE`  or  `INACTIVE`.                                                           |
+| default  | Boolean | `true` for the first instance of this policy, which gets created by default.        |
+
+### Sign On Policy Object Example
+
+```json
+{
+    "id": "rst6v8jmmTIJdv6ms0g4",
+    "name": "Sign On Policy",
+    "type": "Okta:SignOn",
+    "status": "ACTIVE",
+    "default": false,
+    "_links": {
+        "mappings": {
+            "href": "https://idx.okta1.com/api/v1/policies/rst6v8jmmTIJdv6ms0g4/mappings",
+            "hints": {
+                "allow": [
+                    "GET"
+                ]
+            }
+        },
+        "self": {
+            "href": "https://idx.okta1.com/api/v1/policies/rst6v8jmmTIJdv6ms0g4",
+            "hints": {
+                "allow": [
+                    "GET",
+                    "PUT",
+                    "DELETE"
+                ]
+            }
+        },
+        "rules": {
+            "href": "https://idx.okta1.com/api/v1/policies/rst6v8jmmTIJdv6ms0g4/rules",
+            "hints": {
+                "allow": [
+                    "GET"
+                ]
+            }
+        },
+        "deactivate": {
+            "href": "https://idx.okta1.com/api/v1/policies/rst6v8jmmTIJdv6ms0g4/lifecycle/deactivate",
+            "hints": {
+                "allow": [
+                    "POST"
+                ]
+            }
+        },
+        "applications": {
+            "href": "https://idx.okta1.com/api/v1/policies/rst6v8jmmTIJdv6ms0g4/app",
+            "hints": {
+                "allow": [
+                    "GET"
+                ]
+            }
+        }
+    }
+}
+```
+
+## Sign On Rule Object
+
+| Property    | Type                                                                | Description                                                                           |
+|-------------|---------------------------------------------------------------------|---------------------------------------------------------------------------------------|
+| name        | String                                                              | Human-readable name for the Rule, configurable during creation or updating.           |
+| id          | String                                                              | Unique identifier for this Rule (read-only).                                          |
+| type        | String                                                              | Type of the Rule. For Identifier Match Rule Objects, this needs to be `Okta:SignOn`.  |
+| priority    | Integer                                                             | Used to determine which Rules take precedence.                                        |
+| conditions  | Array                                                               | UserType, User, Group and various Device conditions are supported for this rule type. |
```

Comment:
- what's `User` condition?
- looks like we don't support Device in LEA? (I could be wrong but we didn't build in admin UI for LEA)

Reply:
Technically this API support more conditions than what we show in the UI, however I do agree that we should match documentation with what is supported in the UI, so let's change this to: `UserType and Group conditions are supported for this rule type.` and include this in the example below:

```
    "conditions": [
        {
            "key": "Okta:UserType",
            "op": "IN_LIST",
            "value": [
            	"{{userTypeId}}"
        	]
        },
        {
            "key": "Okta:Group",
            "op": "INTERSECTS",
            "value": [
                "{{groupId}}"
            ]
        }	
    ],
```

Reply:
Updated supported conditions for Sign On Rule and added conditions to example object.